Vagrantfile

```diff
@@ -287,18 +290,22 @@ Vagrant.configure('2') do |config|
   end
 
   config.vm.define :f_win2012r2 do |cfg|
-    cfg.vm.box = 'opentable/win-2012r2-standard-amd64-nocm'
+    cfg.vm.box = 'mwrock/Windows2012R2'
     # Without the line below here or in the box, vagrant-omnibus breaks on windows.
     # Reference: https://github.com/chef/vagrant-omnibus/issues/90#issuecomment-51816397
     cfg.vm.guest = :windows
-    default_omnibus config
+    default_omnibus cfg
+    # https://github.com/chef/vagrant-omnibus/issues/118
+    cfg.omnibus.install_url = "https://packages.chef.io/stable/windows/2012r2/chef-client-#{cfg.omnibus.chef_version}-1-x64.msi"
     cfg.vm.provider :virtualbox do |vb|
       vb.customize ['modifyvm', :id, '--memory', 1024]
+      vb.gui = false
     end
     cfg.vm.provision :chef_client do |chef|
       chef_defaults chef, :f_win2012r2, 'splunk_standalone'
       chef.add_role 'splunk_monitors_windows'
       chef.add_recipe 'cerner_splunk'
+      chef.verbose_logging = true
```

Comment:
Actually I'm really not sure why it's in the windows box in particular. That could've been added some time ago. Anyways 9fdb388147a106a667e37d5135d5b9a8b1f5fab7